library/components/TFormattedNumber.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div>{{ formattedNumber }}</div>
```

Comment:
using a div would require changing the layout classes in t-big-number in issues-summary report. I wanted to avoid creating more changes than necessary in topcoat-reports, so decided to change this to a span.